base/osutils.jl

```diff
@@ -56,11 +56,3 @@ end
 @unix_only windows_version() = (0,0)
 
 WINDOWS_XP_VER = (5,1)
```

Comment:
(5,2) is also a Windows XP version (Vista started with  6.0)
https://msdn.microsoft.com/en-us/library/windows/desktop/ms724832%28v=vs.85%29.aspx?f=255&MSPPError=-2147217396

seems like a pre-existing bug, but maybe better to compare against WINDOWS_VISTA_VER?